### ISSUE TITLE
Alternative Card Suit emojis

### ISSUE THREAD

**Issue by Crissov** [Crissov/unicode-proposals#289]
Traditionally, the French card deck uses only two colors for its four suits: red (diamonds and hearts) and black (clubs and spades). For usability and accessibility, there are, however, [card decks with four different colors](https://en.wikipedia.org/wiki/Four-color_deck), one for each suit. Alas, people do not agree about the correct palette.

Card games differ in which order they rank suits, if at all. Unicode code points are ordered in the original English order usually found, e.g., in Bridge spades, heart, diamonds and clubs, but the emoji code points are actually in the now dominant Poker order: spades, clubs, hearts, diamonds and there are [other collations](https://en.wikipedia.org/wiki/Suit_(cards)#Ranking_of_suits).

| Game | Order | Colors |
|------|-------|--------|
| Poker | ♠♣♥♦ | ![](https://upload.wikimedia.org/wikipedia/commons/0/01/Classic_Four_Color_Deck.png) |
| Bridge | ♠♥♦♣ | ![](https://upload.wikimedia.org/wikipedia/commons/b/b0/Bridge_Four_Color_Deck_1.png) ![](https://upload.wikimedia.org/wikipedia/commons/0/07/Seminole_Wars_Deck.png) |
| Preferans, 500 | ♥♦♣♠ | ![]() |
| 99 | ♣♥♠♦ | ![]() |
| Skat | ♣♠♥♦ | ![](https://upload.wikimedia.org/wikipedia/commons/e/ef/Skat_Tournament_Deck.png) |
| Big 2 | ♠♥♣♦ or ♥♦♠♣ | ![]() |
| Teen patti | ♣♥♠♦ | ![]() |
| 13 | ♥♦♣♠ | ![]() |

Historically and locally, the suits have alternate representations and since they are sometimes related to "the" four elements, they are connected to other esoteric, astrological and mystic concepts.

| Suits | Element | Rank | Compass | Zodiac | Colors |
|------|--------|--------|--------|------|------|
| ♣ Clubs = Clovers = Acorns = Batons | Fire ☲ | King ♚ | South | ♈♌♐ |  |
| ♦ Diamonds = Tiles = Bells = Coins = Pentacles = Rings = Crescents | Earth ☷ | Jack ♗ | North | ♉♍♑ | Gold |
| ♠ Spades = Pikes = Wands = Staves = Leaves = Grass = Shields = Swords | Air ☴ | Knight ♞ | East | ♊♎♒ | Green |
| ♥ Hearts = Roses = Cups | Water ☵ | Queen ♕ | West | ♋♏♓ | Red |

Anyway, since all four filled `Black` Unicode emoji suits also have a hollow `White` non-emoji counterpart, these could be used for a different set of colors. Their code points are actually mixed with each other: U+2660&ndash;7 = ♠♡♢♣♤♥♦♧. ♡♢♤♧

**Comment by Crissov** [comment]
Corollary
----------

There probably should be Unicode characters for non-French suits, which could also become emojis:

- ♠
  * 🍂 🍁 ☕🍃 Leaf
  * 🗡 Shield
  * ⚔ 🛡 Sword / Wand #160 / Staff / Pike / Stave
- ♣
  * 🌰 Acorn (#54)
  * 🔨 🏏 🏑 🏒 🎾 🏸 Baton (#67) / Club
  * 🍀 ☘ Clovers
- ♥
  * 🏵 🌹 🌼 🌷 🌻 🌸 🌺 💐 🥀 Rose (flower, blossom)
  * 🏆 🍺 🍷 🍸 🍹 🥃 Cup
- ♦
  * 🏅 🎖 💰 💸 💵 💴 💶 💷 Coin #137 
  * 💎 Diamonds
  * 💍 Rings
  * 🔔 🛎 🎐 Bells
  * 🌛 🌜 🌙 🌘 🌒 🌗 🌓 🌖 🌔 🌕 🌝 🌑 🌚 🎑 Crescents

**Comment by Crissov** [comment]
Mockups
----------

### Twemoji

| Suit | 1 | 2 | 3 | original |
|------|---|---|---|----------|
| Spades | ![2660 blue](https://cloud.githubusercontent.com/assets/6200185/24828439/8ab80a5a-1c5d-11e7-9d71-be8bdb4fdca5.png) | ![2660 green](https://cloud.githubusercontent.com/assets/6200185/24828440/8ab88624-1c5d-11e7-8dd7-eec0e3e57b64.png) | ![2660 cyan](https://cloud.githubusercontent.com/assets/6200185/24828441/8ab9064e-1c5d-11e7-8ec2-88e1cb6d4015.png) | ![2660 black](https://cloud.githubusercontent.com/assets/6200185/24828442/8ab97d90-1c5d-11e7-919e-393c78a98804.png)
| Clubs | ![2663 blue](https://cloud.githubusercontent.com/assets/6200185/24828444/8aca2532-1c5d-11e7-8dbe-d2e09ced8cbe.png) | ![2663 green](https://cloud.githubusercontent.com/assets/6200185/24828445/8accde94-1c5d-11e7-8ed0-57c931066777.png) | ![2663 pink](https://cloud.githubusercontent.com/assets/6200185/24828446/8ad34c02-1c5d-11e7-826c-3174c3799a1d.png) | ![2663 black](https://cloud.githubusercontent.com/assets/6200185/24828443/8ab9dfba-1c5d-11e7-947c-53937d9243ee.png)
| Hearts | ![2665 orange](https://cloud.githubusercontent.com/assets/6200185/24828447/8ad36f7a-1c5d-11e7-8665-eaed95ecb33c.png) | | | ![2665 red](https://cloud.githubusercontent.com/assets/6200185/24828448/8ad5592a-1c5d-11e7-8f51-1cf005699873.png)
| Diamonds | ![2666 orange](https://cloud.githubusercontent.com/assets/6200185/24828450/8add01f2-1c5d-11e7-8701-72c1a8515c25.png) | ![2666 yellow](https://cloud.githubusercontent.com/assets/6200185/24828438/8ab75e2a-1c5d-11e7-8f34-238173cb97f9.png) | ![2666 blue](https://cloud.githubusercontent.com/assets/6200185/24828449/8ad8b2a0-1c5d-11e7-8c24-4cf534b58ad2.png) | ![2666 red](https://cloud.githubusercontent.com/assets/6200185/24828451/8ae0104a-1c5d-11e7-8127-30f6858982f7.png)

**Comment by Porges** [comment]
Swiss decks for Jass have 10 as a 'banner' and Ace as a card with 2 elements on it. (Maybe if additional suits are supported by regional indicators then these could just be variations on the existing characters, <card> + ZWJ + 🇨🇭.)

**Comment by kbjr** [comment]
Haven't seen any activity here in a while, wanted to see if this is something anyone is looking at or working on in any way. I'm currently working on a refresh of an old library of mine for working with decks of cards, and it has support for returning the unicode character corresponding to a particular rank or card, but there are some very clear gaps in what has unicode representation.

So I'm just interested in what if any work is being done in the area.
